src/main/docs/ref/Tags/formActionSubmit.adoc

```diff
@@ -0,0 +1,59 @@
+
+== formActionSubmit
+
+
+
+=== Purpose
+
+
+Creates a submit button that submits to an `formaction` generated by the various link attributes.
+
+
+=== Examples
+
+
+[,xml]
+----
+
+<!--Generates a label of 'Update' submitting to `update` action on the current controller-->
+<g:formActionSubmit value="Update" />
+
+<!--'Update' is action, label is 'Some update label'-->
+<g:formActionSubmit value="Some update label" action="update" />
+
+<!--'Update' is action on the controller 'index', label is 'Some update label'-->
+<g:formActionSubmit value="Some update label" controller="index" action="update" />
+
+<!--label derived from message bundle-->
+<g:formActionSubmit value="${message(code:'label.update')}" action="update" />
+
+<g:formActionSubmit value="Delete" />
+
+<g:formActionSubmit value="DeleteAll"
+                onclick="return confirm('Are you sure???')" />
+
+----
+
+
+=== Description
+
+
+Attributes
+
+* `action` (optional) - The name of the action to use in the formaction; if not specified the default action will be used
+* `controller` (optional) - The name of the controller to use in the formaction; if not specified the current controller will be used
+* `namespace` (optional) - the namespace of the controller to use in the formaction
+* `plugin` (optional) - the name of the plugin which provides the controller
+* `id` (optional) - The element id of the input element
+* `fragment` (optional) - The link fragment (often called anchor tag) to use in the formaction
+* `mapping` (optional) - The {grailsdocs}guide/theWebLayer.html#namedMappings[named URL mapping] to use to rewrite the formaction
+* `params` (optional) - A Map of request parameters for the formaction
+* `url` (optional) - A Map containing the action,controller,id etc for the for the formaction.
```

Comment:
Spaces in list

Reply:
I fixed the documentation that this was copied from as well.